Content.Server/Emp/EmpDisabledComponent.cs

```diff
@@ -0,0 +1,24 @@
+namespace Content.Server.Emp;
+
+/// <summary>
+/// While entity has this component it is "disabled" by EMP.
+/// Add desired behaviour in other systems 
+/// </summary>
+[RegisterComponent]
+[Access(typeof(EmpSystem))]
+public sealed class EmpDisabledComponent : Component
+{
+    /// <summary>
+    /// When this timer hits 0 component will be removed
+    /// </summary>
+    [DataField("timeLeft"), ViewVariables(VVAccess.ReadWrite)]
+    public float TimeLeft;
```

Comment:
Just use a TimeSpan for when it ends, look at other usages of timeoffsetserializer (also sub to entityunpaused + set it to curtime on init if it's < curtime).

Reply:
`TimedDespawnComponent` should use it too I guess, this is where I took it from